playbooks/bootstrap.yml

```diff
@@ -0,0 +1,9 @@
+- name: Install Python on macOS
+  hosts: "{{ macos_hosts | default('all') }}"
```

Comment:
Have you thought about introducing a special inventory subgroup like `macos_12_plus_hosts`?